CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Add renovate package rule for Flipper updates
 - Show all facilities in the state when changing assigned or appointment facility
+- Add a product flavor for Thanks app
```

Comment:
Let's rename the commit and changelog entry to mention Tamil Nadu app instead of Thanks app.